i18n/zh-CN/docusaurus-plugin-content-docs/current/feature/remove-underlying-engine-depdency.md

```diff
@@ -0,0 +1,16 @@
+---
+title: EngineConn模块定义依赖引擎版本
+sidebar_position: 0.2
+---
+
+## 1. 需求背景
+引擎的版本定义默认在 `EngineConn`中，一旦相关版本变更，需要修改多处，我们可以把相关的版本定义统一放到顶层pom文件中
+
+## 2. 使用说明
+编译指定引擎模块时，需要在项目根目录编译，并使用`-pl`来编译具体的引擎模块，比如：
+```
+mvn install package -pl linkis-engineconn-plugins/spark
+
+```
```

Comment:
Can you give an example of how to change the engine version

Reply:
sure ,later will add the example